### PR TITLE
Move cluster helpers to tools subdirectory

### PR DESCRIPTION
**Pull request by kgaillot**
Previously, the unsupported cluster-clean, cluster-helper, and cluster-init tools were kept in the "extra" subdirectory. Now, they're kept in "tools". The fact that they are not installed should be sufficient to indicate that they are unsupported.

A minimal amount of cleaning up of their code is also done, mainly making them .in files so the configured bash path can be used instead of hardcoding /bin/bash, and removing some no-longer-functional options. A lot more could be done, but it's not worth the time.

### CODE REVIEW

**Comment by kgaillot** [comment]
@clumens , could you review this very minor PR when you get a chance? Low priority

File: tools/cluster-clean.in [hunk 1]
```diff
@@ -88,7 +88,7 @@ for f in $state_files; do
 done
 
 cluster-helper $target -- "find /dev/shm -name 'qb-*' -exec rm -f \{\} \;"
-cluster-helper $target -- "find /var/lib/pacemaker/blackbox -name '*-*' -exec rm -f \{\} \;"
+cluster-helper $target -- "find %CRM_BLACKBOX_DIR% -name '*-*' -exec rm -f \{\} \;"
```
Comment (clumens) [inline tools/cluster-clean.in:91]: Did you mean to use `@CRM_BLACKBOX_DIR@` here instead?

File: tools/cluster-init.in [hunk 2]
```diff
@@ -579,12 +563,6 @@ for host in $host_list; do
...
 
-    if [ $pcmk_ver = 10 ]; then
-	scp /etc/hosts root@${host}:/etc/hosts
-	scp ~/.ssh/id_dsa.suse root@${host}:.ssh/id_dsa
-	scp ~/.ssh/known_hosts root@${host}:.ssh/known_hosts
-    fi
-
     ssh -l root ${host} -- grep -q "token:" $cs_conf 2>&1 > /dev/null
     new_config=$?
     new_config=1
```
Comment (clumens) [inline tools/cluster-init.in:528]: Typo in commit message - installed -> install

File: tools/cluster-init.in [hunk 3]
```diff
@@ -365,15 +334,6 @@ esac
...
-	    info Configuring Clusterlabs repo: $REPO
-	    yum install -y wget
-	    rm -f /etc/yum.repos.d/clusterlabs.repo
-            wget -O /etc/yum.repos.d/clusterlabs.repo https://www.clusterlabs.org/$REPO/clusterlabs.repo &>/dev/null
-	    yum clean all
-	fi
-
 	info Installing cluster software
 	yum install -y $pkgs pacemaker
 	;;
```
Comment (clumens) [inline tools/cluster-init.in:339]: Same typo in commit message for this one, too.

File: tools/cluster-init.in (hunk 2 above)
Comment (kgaillot) [reply tools/cluster-init.in:528]: Thanks, fixed

**Comment by kgaillot** [comment]
Updated commits per review

**Pull request merged**